rsrcdump/extract.py

```diff
@@ -38,7 +46,7 @@ def extract_resource_map(res_map, outpath, include_types=[], exclude_types=[], q
             else:
                 obj = res.data
 
-            wrapper = {}
+            wrapper: Dict[str, Union[int, str]] = {}
```

Comment:
```suggestion
            wrapper: dict[str, Any] = {}
```
I'd like to avoid maintaining a complex signature for all types that can be returned by the ResourceConverters.
For example, StringListConverter returns a list[str], which isn't covered by Union[int,str].
So, how about just using dict[str, Any]?

Reply:
I guess that works